docker/bitmask-docker.sh

```diff
@@ -81,7 +81,9 @@ help() {
     echo ">> Bitmask on docker"
     echo "Run the bitmask app in a docker container."
     echo
-    echo "Usage: $0 {init bitmask.json | update | run | help }"
+    echo "Usage: $0 {init | update} [bitmask-nightly.json] {run | help}"
```

Comment:
I think that this suggest that using: `./bitmask-docker.sh bitmask.json run` is a valid command.
If you want to reuse the json mandatory parameter it should be something like this:
```
"Usage: $0 {(init | update) bitmask-nightly.json} {run | help}"
```
Maybe the `{}` are not needed.

I think that it's more clear to specify this as:
```
"Usage: $0 {init bitmask.json | update bitmask.json | run | help}"
```